impeller/renderer/backend/vulkan/test/mock_vulkan_unittests.cc

```diff
@@ -0,0 +1,37 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/testing/testing.h"  // IWYU pragma: keep
+#include "gtest/gtest.h"
+#include "impeller/renderer/backend/vulkan/command_pool_vk.h"
+#include "impeller/renderer/backend/vulkan/test/mock_vulkan.h"
+
+namespace impeller {
+namespace testing {
+
+TEST(MockVulkanContextTest, IsThreadSafe) {
```

Comment:
Is the idea that randomly this would crash if it wasn't threadsafe?  By it's nature MockVulkanContext only lives to be run on tests so there is no need to add tests to it.  If it was helpful to developing the change I'm fine with keeping the test if you want.

Reply:
Yeah it was entirely to isolate the problems and probably useful to keep around.